### PR TITLE
AMB-1003 test egress rules

### PR DESCRIPTION
**Pull request by Valentinolucas**
## Summary
Testing egress rules on AWS hosted target infra

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: tests/test_api.py [hunk 1]
```diff
@@ -19,12 +19,11 @@ def url(self, apigee_token) -> str:
...
     @pytest.mark.debug
-    @pytest.mark.skip(
-        reason="This tests must be skipping for now, to avoid errors on the pipeline"
-    )
     def test_happy_path(self, apigee_token, url):
         # Given
-        url = "https://internal-dev.api.service.nhs.uk/gp-connect-access-record-pr-40/test"
+        # url = "https://internal-dev.api.service.nhs.uk/gp-connect-access-record-pr-47/AllergyIntolerance"
+        # url = "https://internal-dev.api.service.nhs.uk/gp-connect-access-record-pr-47/test"
+        url = "http://localhost:9000/test"
```
Comment (Valentinolucas) [inline tests/test_api.py:26]: dont forget to point this back to INT

File: tests/test_api.py [hunk 2]
```diff
@@ -9,43 +7,38 @@ class TestAllergyIntolerance:
...
-        reason="This tests must be skipping for now, to avoid errors on the pipeline"
-    )
-    def test_happy_path(self, apigee_token, url):
+    def test_happy_path(self, access_token, url):
         # Given
-        url = "https://internal-dev.api.service.nhs.uk/gp-connect-access-record-pr-40/test"
-        token = apigee_token
         expected_status_code = 200
+        expected_content = "B82617"
+        print(url)
```
Comment (Valentinolucas) [inline tests/test_api.py:20]: can this print statement go away?

**Review by urchy — APPROVED** [review]

**Pull request merged**